packages/custom-field/test/visual/lumo/custom-field.test.js

```diff
@@ -14,6 +14,7 @@ import '@vaadin/text-area/theme/lumo/vaadin-text-area.js';
 import '@vaadin/text-field/theme/lumo/vaadin-text-field.js';
 import '@vaadin/polymer-legacy-adapter/template-renderer.js';
 import '@vaadin/time-picker/theme/lumo/vaadin-time-picker.js';
+import '@vaadin/rich-text-editor/theme/lumo/vaadin-rich-text-editor.js';
```

Comment:
Using the rich text editor here is a bit much IMO, and could lead to false positives in case something in the RTE changes. Alternatively you could just use a native input field.

Reply:
I agree that native `input` is better.

Reply:
Ping @sissbruecker, done